packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -237,7 +266,41 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
 
   @override
   Widget build(BuildContext context) {
-    final TextStyle textStyle = Theme.of(context).textTheme.caption;
+    // TODO(ianh): This whole build function doesn't handle RTL yet.
+    ThemeData theme = Theme.of(context);
```

Comment:
We're inconsistent about how we name this variable.  Current thought is to use `themeData`.